base/deprecated.jl

```diff
@@ -924,14 +924,19 @@ for f in (
         :erfinc, :erfcinv, # base/special/erf.jl
         :trunc, :floor, :ceil, :round, # base/floatfuncs.jl
         :rad2deg, :deg2rad, :exponent, :significand, # base/math.jl
-        :unix2datetime, :rata2datetime, :julian2datetime, # base/dates/conversions.jl
         :sind, :cosd, :tand, :asind, :acosd, :atand, :asecd, :acscd, :acotd, # base/special/trig.jl
         )
     @eval @dep_vectorize_1arg Real $f
 end
 # base/complex.jl
 @dep_vectorize_1arg Complex round
 @dep_vectorize_1arg Complex float
+# base/dates/*.jl
+for f in (
+        :unix2datetime, :rata2datetime, :julian2datetime, # base/dates/conversions.jl
+        )
+    eval(Dates, :(Base.@dep_vectorize_1arg Real $f))
+end
```

Comment:
IMHO those parentheses make it kind of hard to read. Might be nicer as
```julia
# base/dates/*.jl
for f in (:unix2datetime, :rata2datetime, :julian2datetime) # base/dates/conversions.jl
    eval(Dates, :(Base.@dep_vectorize_1arg Real $f))
end
```
If you were worried about line lengths, this is actually shorter. :smile: